lib/dependencies/ExportsInfoDependency.js

```diff
@@ -46,6 +46,12 @@ const getProperty = (moduleGraph, module, exportName, property, runtime) => {
 		}
 	}
 	switch (property) {
+		case "canMangle": {
+			const exportsInfo = moduleGraph.getExportsInfo(module);
+			const exportInfo = exportsInfo.getExportInfo(exportName);
+			if (exportInfo) return exportInfo.canMangle;
+			return exportsInfo.otherExportsInfo.canMangleProvide;
```

Comment:
Why is that `canMangleProvide` and not `canMangle`?

```suggestion
			return exportsInfo.otherExportsInfo.canMangle;
```